.github/workflows/ansible.yml

```diff
@@ -7,24 +7,38 @@ on:
         type: string
         required: true
 jobs:
-  lint:
-    uses: famedly/github-workflows/.github/workflows/ansible-lint.yml
+  lint-roles:
+    name: "Run ansible-lint"
+    uses: famedly/github-workflows/.github/workflows/ansible-lint.yml@sammy/ansible-improvements
     with:
       collection: ${{ inputs.collection }}
-  test-units:
-    uses: famedly/github-workflows/.github/workflows/ansible-test.yml
+      target: roles
+  lint-playbooks:
+    name: "Run ansible-lint"
+    uses: famedly/github-workflows/.github/workflows/ansible-lint.yml@sammy/ansible-improvements
     with:
       collection: ${{ inputs.collection }}
+      target: playbooks
+  test-units:
+    name: "Run unit tests"
+    uses: famedly/github-workflows/.github/workflows/ansible-test.yml@sammy/ansible-improvements
+    with:
       testing-type: units
+      collection: ${{ inputs.collection }}
   test-sanity:
-    uses: famedly/github-workflows/.github/workflows/ansible-test.yml
+    name: "Run sanity tests"
+    uses: famedly/github-workflows/.github/workflows/ansible-test.yml@sammy/ansible-improvements
     with:
-      collection: ${{ inputs.collection }}
       testing-type: sanity
+      collection: ${{ inputs.collection }}
   test-integration:
-    uses: famedly/github-workflows/.github/workflows/ansible-test.yml
+    name: "Run integration tests"
+    uses: famedly/github-workflows/.github/workflows/ansible-test.yml@sammy/ansible-improvements
     with:
-      collection: ${{ inputs.collection }}
       testing-type: integration
+      collection: ${{ inputs.collection }}
   black:
-    uses: famedly/github-workflows/.github/workflows/black.yml
+    name: "Format using black"
+    uses: famedly/github-workflows/.github/workflows/black.yml@sammy/ansible-improvements
```

Comment:
This, and every other occurance of it, needs to be fixed

Reply:
whoops, fixed